srml/council/src/motions.rs

```diff
@@ -76,7 +120,8 @@ decl_module! {
 			ensure!(!<ProposalOf<T>>::exists(proposal_hash), "duplicate proposals not allowed");
 
 			if threshold < 2 {
-				let ok = proposal.dispatch(Origin::Members(1).into()).is_ok();
+				let seats = <Council<T>>::active_council().len() as MemberCount;
+				let ok = proposal.dispatch(RawOrigin::Members(1, seats).into()).is_ok();
```

Comment:
Why do we dispatch here with `Members(...)` and not with `Member(who)`? Should we dedicate `Members(n, _)` only to cases where n > 1?

Reply:
it's possible, but i didn't see any reason to special-case n==1 (or even n==0).

Reply:
I am worried a little bit about receivers of this origin: if it is enough to have only a single approver then the receiver will have to match against Member(_) and Members(1, _), if I understood the idea correctly. Although in this case we need to maintain this additional n>1 invariant...

Reply:
What counts as "enough" is entirely at the discretion of the code. There's nothing special about `Origin::Members(2)` that makes it a fundamentally different beast to `Origin::Members(1)`.

Reply:
(`Member` is provided as it has slightly different semantics - it gives the `AccountId` of the member as its `Success`; `Members` gives you the number of members matched and the total size as its `Success` - in some circumstances it could theoretically be more useful to require a Members(1) origin over a Member origin. In that case any member that wanted to do so would need to use the relevant dispatch function.)